internal/provider/resource_google_cloud_provider_credential.go

```diff
@@ -0,0 +1,95 @@
+package provider
+
+import (
+	"context"
+
+	"github.com/PacketFabric/terraform-provider-packetfabric/internal/packetfabric"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceCloudProviderCredentialGoogle() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCloudProviderCredentialGoogleCreate,
+		ReadContext:   resourceCloudProviderCredentialRead,
+		UpdateContext: resourceCloudProviderCredentialGoogleUpdate,
+		DeleteContext: resourceCloudProviderCredentialDelete,
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+				Description:  "Description of the Cloud Provider Credentials.",
+			},
+			"service_account": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Same as with the AWS params above

Reply:
I just realized it should be also `google_service_account`!